glean-core/csharp/Glean/Glean.csproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What changed on this line?

Reply:
Um... I don't know?  Line ending maybe?  This is weird.

Reply:
That doesn't show up as a change on my local diff, just the two Package Refs below. Maybe something weird with GitHub?